frontend/interfaces/schedulable_query.ts

```diff
@@ -0,0 +1,110 @@
+import { IFormField } from "./form_field";
+import { IPack } from "./pack";
+
+// Query itself
+export interface IScheduleableQuery {
+  created_at: string;
+  updated_at: string;
+  id: number;
+  name: string;
+  description: string;
+  query: string;
+  team_id: number | null;
+  interval: number;
+  platform: string; // TODO: Specify strings
```

Comment:
This looks like we can use `IPlatformString` from `frontend/interfaces/platform.ts`